libraries/helpers.rb

```diff
@@ -220,6 +220,21 @@ def active_directory_from_resource
       base_resource(new_resource, spec, 'authentication/v2')
     end
 
+    def auth_oidc_from_resource
+      spec = {}
+      spec['additional_scopes'] = new_resource.additional_scopes if new_resource.additional_scopes
+      spec['client_id'] = new_resource.client_id
+      spec['client_secret'] = new_resource.client_secret
+      spec['disable_offline_access'] = new_resource.disable_offline_access
```

Comment:
I checked the spec and we should match it but I really dislike boolean values with a negative assertion in its name as it forces you to think in terms of double negatives.
```
# these are immediately understandable
ofline_access = true
offline_access = false
disable_offline_accesss = true

# I always find myself taking a few extra seconds to logic this in my head
disable_offline_accesss = false

Reply:
Definitely agree. Just stuck with following upstream so we'd match here.